grpc/README.md

```diff
@@ -9,3 +9,26 @@ from GRPC, and work with both the Protobuf and FlatBuffers code generator.
 the GRPC libraries for this to compile. This test will build using the
 `FLATBUFFERS_BUILD_GRPCTEST` option to the main FlatBuffers CMake project.
 
+## Building Flatbuffers with gRPC
+
+### Linux
+
+1. Build and install gRPC. See [instructions](https://github.com/grpc/grpc/tree/master/src/cpp).
+2. Build and install protobuf. See [instructions](https://github.com/protocolbuffers/protobuf/blob/master/src/README.md).
```

Comment:
Does step 2 not happen automatically as a consequence of 1? I never remember building protobuf manually.

Reply:
No, it was not automatic. I had to do two separate build and install. I'll give it a shot on a different Linux distro.